ruleset/rules/0808-attack-TA0008.xml

```diff
@@ -10,8 +10,70 @@
+        <mitre>
+            <id>T1570</id>
+        </mitre>
+    </rule>
+
     <!-- Sample: {"win":{"eventdata":{"originalFileName":"PSCP","image":"C:\\\\Users\\\\AtomicRed\\\\AppData\\\\Roaming\\\\TransbaseOdbcDriver\\\\pscp.exe","product":"PuTTY suite","parentProcessGuid":"{4dc16835-3bdd-609c-5901-000000003b00}","description":"Command-line SCP/SFTP client","logonGuid":"{4dc16835-2e5f-609c-19a4-7c0000000000}","parentCommandLine":"C:\\\\Windows\\\\system32\\\\cmd.exe","processGuid":"{4dc16835-416a-609c-7601-000000003b00}","logonId":"0x7ca419","parentProcessId":"5560","processId":"2180","currentDirectory":"C:\\\\Users\\\\AtomicRed\\\\AppData\\\\Roaming\\\\TransbaseOdbcDriver\\\\","utcTime":"2021-05-12 20:58:18.149","hashes":"SHA1=EE1B5F3A7F9563A9E7575EDDA467794584555F97,MD5=3145D4A197A3523253880E9E6E76F798,SHA256=7A09FD6885CA5662A40D6F9212C115D8F38C88F9C1BCDA697854BFB202F289B2,IMPHASH=C2612378E2F461D6FCF8743722ABEB7A","parentImage":"C:\\\\Windows\\\\System32\\\\cmd.exe","ruleName":"technique_id=T1059.003,technique_name=Windows Command Shell","company":"Simon Tatham","commandLine":"pscp.exe  -scp psexec.py administrator@Exchangetest.com@192.168.0.218:/tmp/psexec.py","integrityLevel":"Medium","fileVersion":"Release 0.73","user":"EXCHANGETEST\\\\AtomicRed","terminalSessionId":"1"},"system":{"eventID":"1","keywords":"0x8000000000000000","providerGuid":"{5770385f-c22a-43e0-bf4c-06f5698ffbd9}","level":"4","channel":"Microsoft-Windows-Sysmon/Operational","opcode":"0","message":"\"Process Create:\r\nRuleName: technique_id=T1059.003,technique_name=Windows Command Shell\r\nUtcTime: 2021-05-12 20:58:18.149\r\nProcessGuid: {4dc16835-416a-609c-7601-000000003b00}\r\nProcessId: 2180\r\nImage: C:\\Users\\AtomicRed\\AppData\\Roaming\\TransbaseOdbcDriver\\pscp.exe\r\nFileVersion: Release 0.73\r\nDescription: Command-line SCP/SFTP client\r\nProduct: PuTTY suite\r\nCompany: Simon Tatham\r\nOriginalFileName: PSCP\r\nCommandLine: pscp.exe  -scp psexec.py administrator@Exchangetest.com@192.168.0.218:/tmp/psexec.py\r\nCurrentDirectory: C:\\Users\\AtomicRed\\AppData\\Roaming\\TransbaseOdbcDriver\\\r\nUser: EXCHANGETEST\\AtomicRed\r\nLogonGuid: {4dc16835-2e5f-609c-19a4-7c0000000000}\r\nLogonId: 0x7CA419\r\nTerminalSessionId: 1\r\nIntegrityLevel: Medium\r\nHashes: SHA1=EE1B5F3A7F9563A9E7575EDDA467794584555F97,MD5=3145D4A197A3523253880E9E6E76F798,SHA256=7A09FD6885CA5662A40D6F9212C115D8F38C88F9C1BCDA697854BFB202F289B2,IMPHASH=C2612378E2F461D6FCF8743722ABEB7A\r\nParentProcessGuid: {4dc16835-3bdd-609c-5901-000000003b00}\r\nParentProcessId: 5560\r\nParentImage: C:\\Windows\\System32\\cmd.exe\r\nParentCommandLine: C:\\Windows\\system32\\cmd.exe\"","version":"5","systemTime":"2021-05-12T20:58:18.1529188Z","eventRecordID":"199658","threadID":"3320","computer":"hrmanager.ExchangeTest.com","task":"1","processID":"2080","severityValue":"INFORMATION","providerName":"Microsoft-Windows-Sysmon"}}} -->
-    <rule id="92080" level="6">
+    <rule id="92105" level="6">
```

Comment:
Why is a rule id change?

Reply:
There was rule id collision in sequential numbering in other file